core/src/main/java/org/apache/spark/api/java/function/DoubleFunction.java

```diff
@@ -23,5 +23,5 @@
  *  A function that returns Doubles, and can be used to construct DoubleRDDs.
  */
 public interface DoubleFunction<T> extends Serializable {
-  public double call(T t) throws Exception;
+  double call(T t) throws Exception;
```

Comment:
Thank you for you review. Yes, this PR removes unnecessary keywords due to their default meaning in Java language. Also, I agree with you that sometimes *explicitly* written code (e.g. *interface method*) also is good for emphasizing the meaning.